feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/inc/nl80211_cpy.h

```diff
@@ -0,0 +1,6930 @@
+#ifndef __LINUX_NL80211_H
```

Comment:
Why make a copy , instead patch the nl80211.h

Reply:
The copy is needed because the standard one is part of toolchain which we do not to mess with. That is the reason different modules keep a copy of it.